russian/light-fire/sing/marshrutka/marshrutka.md

```diff
@@ -1 +1,3 @@
 И через пару минут на горизонте, и правда, появилась маршрутка №1 времен CCCР.
+
+В маршрутке нет водителя, но вы садитесь в неё, дверь автоматически закрывается и маршрутка увозит вас из этого места. По дороге вы видите красивые леса, поля и какие-то деревеньки. Постепенно вы засыпаете.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.